### ISSUE TITLE
Error 403 when pulled from git (my own project)

### ISSUE THREAD

**Issue by tdserviceco** [themosis/framework#243]
When I'm start from a previous project (via sourcetree) and pull it down..
And when I change to what I need to change (enviroment.php & .env.local.php) I get a 403 error
![skarmavbild 2015-12-08 kl 22 11 18](https://cloud.githubusercontent.com/assets/10770786/11668657/b124ce06-9df8-11e5-9aee-9af07a201cb7.png)
How comes? is there a fix? (for now It works on my other computer) I cant share my project with my partner/co-worker or friend..

**Comment by jlambe** [comment]
Probably a permission issue on your computer or missing rule on your vhost. Check that your group/user has access to your project.

**Comment by tdserviceco** [comment]
well thats the funny part.. I copy and paste in Vhost what i used before.. 
```Vhost
 DocumentRoot /Applications/MAMP/htdocs/Privat/themosis-demo/htdocs
        <Directory /Applications/MAMP/htdocs/Privat/themosis-demo/htdocs>
                Options FollowSymLinks
                AllowOverride All
        </Directory>
Now I get a error 500...

**Comment by jlambe** [comment]
Can you check the log files from your Mamp application both from Apache and PHP so perhaps there is a hint to what's causing the issue.

**Comment by tdserviceco** [comment]
Found the issue.. but found another one.. when I download a new project from composer and choose Swedish (as an example).. and I get this:
![skarmavbild 2015-12-09 kl 09 37 27](https://cloud.githubusercontent.com/assets/10770786/11680435/874081d8-9e58-11e5-992f-6a281387bec9.png)

**Comment by jlambe** [comment]
Glad it's working.

For the other issue, I've already submitted a ticket on WordPress core (as it is one from WordPress) here:
 - https://core.trac.wordpress.org/ticket/34164

The issue has a patch and apparently will be merged into the next WordPress release 4.5. During installation, just turn off error reporting to 0 from your `config/environments/local.php` file then put it back to `-1` once WordPress is installed.
